### PR TITLE
Cloud hosted adjustments

### PR DESCRIPTION
**Pull request by davidwyao**

### CODE REVIEW

**Review by gkwan-ibm — COMMENTED** [review]
@davidwyao 
- at line 84, and 198 " to specify the location of **artists.json** on the cloud:" is not clear enough
- at line 151, `cd /home/projects/...` -> `cd /home/project/...`
- at line 165-168, suggest to use curl command
- at line 200, use full path of consume-rest.js instead starting from the src directory
- at line 301, try add a backtick at "**`http"
  - and "ai`**", so the no direct link is shown on the lab

File: README.adoc [hunk 1]
```diff
@@ -57,11 +64,59 @@ https://openliberty.io/guides/rest-intro.html[Creating a RESTful web service^].
...
+To try out the application, first go to the `finish` directory and run the following
+Maven goal to build the application and deploy it to Open Liberty:
+```
+cd finish
+mvn liberty:run
+```
+endif::[]
+
+ifdef::cloud-hosted[]
+In this IBM cloud environment, you need to update the URL to access the **artists.json** in the **consume-rest.js** file.
```
Comment (dmuelle) [inline README.adoc:82]: generally, we use `monospace` for file names and **bold** for UI elements:

you need to update the URL to access the `artists.json` in the `consume-rest.js` file.

....go to the `finish` directory and update the `consume-rest.js` file:

File: README.adoc [hunk 2]
```diff
@@ -117,6 +190,14 @@ consume-rest.js
 include::finish/src/main/webapp/js/consume-rest.js[]
 ----
 
+ifdef::cloud-hosted[]
+Run the following command to update the URL to access the **artists.json** in the **consume-rest.js** file:
```
Comment (dmuelle) [inline README.adoc:194]: see previous comment re monospace for file names

File: README.adoc [hunk 3]
```diff
@@ -193,9 +286,17 @@ dj wrote 0 albums:
...
 No explicit code directly uses the consumed artist JSON, so you do not need to write any test cases for this guide.
 
+ifndef::cloud-hosted[]
 Whenever you change your AngularJS implementation, the application root at http://localhost:9080[http://localhost:9080^] will
 reflect the changes automatically. You can visit the root to manually check whether the artist JSON
 was consumed correctly.
+endif::[]
+
+ifdef::cloud-hosted[]
+Whenever you change your AngularJS implementation, the application root at **`http://accountname-9080.theiadocker-4.proxy.cognitiveclass.ai`** will
```
Comment (dmuelle) [inline README.adoc]: don't need bold for the URL

**Review by dmuelle — CHANGES_REQUESTED** [review]
Looks good- just a few comments on formatting syntax

File: README.adoc (hunk 1 above)
Comment (gkwan-ibm) [reply README.adoc:82]: unfortunately, SN(cloud-hosted) cannot show monospace nicely, so we use bold

Comment (dmuelle) [reply README.adoc:82]: ok- then disregard comments to that effect

File: README.adoc (hunk 3 above)
Comment (gkwan-ibm) [reply README.adoc]: addressed

**Review by dmuelle — APPROVED** [review]
LGTM

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**